src/neptune/internal/threading/daemon.py

```diff
@@ -125,12 +132,21 @@ def wrapper(self_: Daemon, *args, **kwargs):
                         return result
                     except NeptuneConnectionLostException as e:
                         if self_.last_backoff_time == 0:
-                            logger.warning(
-                                "Experiencing connection interruptions."
-                                " Will try to reestablish communication with Neptune."
-                                " Internal exception was: %s",
-                                e.cause.__class__.__name__,
-                            )
+                            if e.cause.__class__.__name__ == "HTTPTooManyRequests":
+                                warn_once(
+                                    "Looks like you're reaching the default workspace logging-rate limit."
```

Comment:
```suggestion
                                    "You're hitting the default logging-rate limit for your workspace."
```
Sorry, missed this one 😅

Reply:
@szaganek - as a penalty, you'll have to approve again 😆